inc/class-command.php

```diff
@@ -255,4 +256,30 @@ protected function secure( InputInterface $input, OutputInterface $output ) {
 	protected function shell( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( 'vagrant ssh' );
 	}
+
+	/**
+	 * Get the name of the project host names from the local config.
+	 *
+	 * @return array
+	 */
+	protected function get_project_hosts() : array {
+
+		$composer_json = json_decode( file_get_contents( getcwd() . '/composer.json' ), true );
```

Comment:
Can we use `Altis\get_config()` directly? Think it might be loaded via autoload already 🤔

Reply:
Hmm. In not super confident that's the case. I borrowed this from the `local-server` implementation

Reply:
Or at least we shouldn't use it unless we make `altis/core` a dependency

Reply:
I think we can make it a dep easily enough; the problem with not using it is that we'll potentially have differences between our regular parsing and here; e.g. env-specific configuration doesn't work the same - obviously not a huge concern, but the inconsistency is harmful.

Reply:
`Altis/get_config()` isn't defined when the composer command is run. Also this config only really relates to local so I think we have to accept the limitation re. parsing for env specific config.

Reply:
Rats, oh well then!